WikiDrips/UIImageExtensions.swift

```diff
@@ -10,15 +10,15 @@ import UIKit
 
 extension UIImage {
     
-    func image(withInitials: String?, in bounds: CGRect) -> UIImage {
+    static func image(withInitials initials: String?, in bounds: CGRect) -> UIImage {
         let scale = UIScreen.main.scale
         
         UIGraphicsBeginImageContextWithOptions(bounds.size, false, scale)
         defer {
             UIGraphicsEndImageContext()
         }
         
-        guard let initials = withInitials, let context = UIGraphicsGetCurrentContext() else {
+        guard let monogram = initials, let context = UIGraphicsGetCurrentContext() else {
```

Comment:
Ah!  I misread the error I was getting when I initially tried to do initials = initials.  I had my method arg and its label reversed.  So when it wouldn't compile, I jumped to the wrong conclusion and thought guard made local variables scope differently.  Anyway, thanks for the explanation.